projects/canopy/src/lib/primary-message/primary-message.component.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { TestBed, waitForAsync } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+
+import { MockComponents, MockedComponentFixture, MockModule, MockRender } from 'ng-mocks';
+
+import {
+  LgPrimaryMessageComponent,
+  LgPrimaryMessageDescriptionComponent,
+  LgPrimaryMessageTitleComponent,
+} from './';
+import { LgBrandIconModule } from '../brand-icon';
+
+describe('LgPrimaryMessageComponent', () => {
+  let component: LgPrimaryMessageComponent;
+  let fixture: MockedComponentFixture<LgPrimaryMessageComponent>;
+
+  beforeEach(
+    waitForAsync(() => {
+      TestBed.configureTestingModule({
+        declarations: [
+          LgPrimaryMessageComponent,
+          MockComponents(
+            LgPrimaryMessageTitleComponent,
+            LgPrimaryMessageDescriptionComponent,
+          ),
+        ],
+        imports: [MockModule(LgBrandIconModule)],
+      }).compileComponents();
+    }),
+  );
+
+  beforeEach(() => {
+    fixture = MockRender(`
+      <lg-primary-message [iconName]="iconName" [hasRole]="hasRole">
```

Comment:
are these inputs initially passed in as undefined?

Reply:
is icon name an input?

Reply:
Oh i need to remove `iconName`. 
`hasRole` is passed as `true` by default.

Reply:
Removed both of the inputs from the `MockRender` :)